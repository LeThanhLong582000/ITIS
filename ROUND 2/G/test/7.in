25
100110100110110
01110100111111000111100101111011110101001101111001111001
11001111010100001111001001011110100
001110111001001100001111011000000001111111100101100101111
010010111011111001010100110001011001
01111011000100110000110
010010001001101101110010010000111101011011101110010011000001001000001100110100111000111000000011
0001000100101111011
010110010010110000110101
000100000101011100110100011100011101110
00010010010010010110100011101111001010100101000
0010110111100010001110001100010110011111011111001000111110001100111100000010010111100111001110
11011000010101000100010100000100100001110110101000001011110110010001100111001010110010010101110100
101110010010
0101010110101011001101101010011100111011100100111101101101101100001111101000010101001
011010011011100101100001111111000000010111001
101111100001010111001110011110101011111011101000100001101100110001110110101
110100101010110000
01011101110010
00011111000111101101110111110010010000110001101011101100000011
00101010010000110010011010111010011100101011011110011000110011110000101111011110010010
01010001110010101001001001011100001011010111111000101
000010011100101110110000110110111110101101100111000100101010110111001101100
0010011101110010111001110101001101100111101010011110110000101001111100100011111000011010100001
0001010100010110
