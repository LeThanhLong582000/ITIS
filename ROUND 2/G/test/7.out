6
19
14
21
14
7
40
4
9
14
19
41
40
3
34
17
30
6
4
23
35
21
30
39
5
